content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/kubelet-credential-providers.md

```diff
@@ -0,0 +1,29 @@
+---
+title: KubeletCredentialProviders
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+
+stages:
+  - stage: alpha 
+    defaultValue: false
+    fromVersion: "1.20"
+    toVersion: "1.23"
+  - stage: beta 
+    defaultValue: true
+    fromVersion: "1.24"
+    toVersion: "1.25"
+  - stage: stable
+    defaultValue: true
+    fromVersion: "1.26"
+    toVersion: "1.28"
+
+removed: true
+---
+
+<!--
+Enable kubelet exec credential providers for
+image pull credentials.
+-->
+启用 kubelet exec 凭证提供者以获取镜像拉取凭证。
```

Comment:
```suggestion
启用 kubelet exec 凭据提供程序以获取镜像拉取凭据。
```
Refer to: https://kubernetes.io//zh-cn/docs/tasks/administer-cluster/kubelet-credential-provider/